src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -901,6 +901,74 @@ BOOST_AUTO_TEST_CASE(GradientFastLSTMLayerTest)
   BOOST_REQUIRE_LE(CheckGradient(function), 0.2);
 }
 
+/**
+ * Testting the overloaded Forward of LSTM Layer.
```

Comment:
Testing and what about we write to the cell state.

Reply:
Sorry, I just forgot that. I am adding it.